addons/sourcemod/translations/vip_core.phrases.txt

```diff
@@ -2,30 +2,27 @@
 {
 	"VIP_CHAT_PREFIX"
 	{
-		"en"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"fi"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"de"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"pt"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"pt_p"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"ru"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"ua"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
+		"en"		"[VIP]"
```

Comment:
do not replace color placeholders with codes, because they may differ in different games